### PR TITLE
Set up cross validation script

### PR DESCRIPTION
**Pull request by murphymadeleine21**

### CODE REVIEW

File: syserol/model.py [hunk 1]
```diff
@@ -51,7 +51,7 @@ def cross_validation():
         matrix[test_index, 0:6] = copy[test_index, len(glycan) : len(glycan) + 6]  # store original value
         copy[test_index, len(glycan) : len(glycan) + 6] = np.nan  # artificially make the value NaN
 
-        _, matrixFac, _ = perform_CMTF(cube, copy, 2)  # run decomposition on new matrix
+        _, matrixFac, _ = perform_CMTF(cube, copy, 10)  # run decomposition on new matrix
```
Comment (aarmey) [inline syserol/model.py]: Make the number of components an argument to the function. You may want to run this for varied numbers.

File: syserol/model.py [hunk 2]
```diff
@@ -37,7 +37,7 @@ def test_predictions(function="ADCD"):
 
 
 def cross_validation():
```
Comment (aarmey) [inline syserol/model.py]: Need a better name.

**Review by aarmey — CHANGES_REQUESTED** [review]

**Comment by murphymadeleine21** [comment]
We could also potentially have the script run with all of the different component numbers and write out all of the files? But that would take a very long time..

**Comment by aarmey** [comment]
Not for now.

**Comment by murphymadeleine21** [comment]
I'm pretty sure I addressed all of the changes you wanted for this PR, if we want to merge.

**Review by aarmey — APPROVED** [review]

**Pull request merged**